ethereum_history_api/oracles/src/noir/noir_js/encode.ts

```diff
@@ -18,6 +18,16 @@ export function encodeHexStringToArray(value: string): Uint8Array {
   return new Uint8Array(arr);
 }
 
+export function encodeNullable(value: string | null): string {
+  return 'Option::' + (value !== null ? `some(${value})` : 'none()');
```

Comment:
```suggestion
  if (value !== null) {
    return `Option::some(${value})`;
  } else {
    return 'Option::none()';
  }
```

I feel like in this case - the if solution has more lines, but is more readable. Please let me know if you agree

Reply:
Super nit. No strong feelings here

Reply:
changed